scripts/start_port.sh

```diff
@@ -62,12 +62,22 @@ git add "$mathlib4_path"
 git commit -m 'Initial file copy from mathport'
 
 sed -i 's/Mathbin\./Mathlib\./g' "$mathlib4_path"
+sed -i '/^import/{s/[.]Smul/.SMul/g; s/[.]Pnat/.PNat/g}' "$mathlib4_path"
+
+# awk script taken from https://github.com/leanprover-community/mathlib4/pull/1523
+awk '{do {{if (match($0, "^  by$") && length(p) < 98) {p=p " by";} else {if (NR!=1) {print p}; p=$0}}} while (getline == 1) if (getline==0) print p}' "$mathlib4_path"
```

Comment:
In general, I think just running this awk command will give you the replaced file on stdout. Perhaps I'm not understanding something, but probably try if it works at written. 
If you want to replace it, then try either add `-i -inplace` or write it to a temporary fie with `> tmpfile` and then copy it.

Reply:
thanks, done